### DISCUSSION TITLE
What does the `enableVirtualization` do?

### DISCUSSION THREAD

**Discussion by mr-rexestremera** [Comcast/react-data-grid#3289] [Q&A]
Apparently it causes the grid to NOT render all the column when running jest. So now my question are as follows:

* What does the `enableVirtualization` do? 
* does it improve performance? 
* Is there any difference if i set it to false?

**Comment by nstepien** [comment]
`enableVirtualization` is enabled by default. When virtualization is enabled, only cells within the viewport are rendered to optimize render times. This is what allows the grid to remain smooth even when the grid has millions of cells.
